docker-for-mac/release-notes.md

```diff
@@ -11,6 +11,31 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 3.3.2
+2021-04-29
+
+[Mac with Intel chip](?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
+[Mac with Apple chip](?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
+
+## Upgrades
+
+- [Compose CLI v1.0.14](https://github.com/docker/compose-cli/tree/v1.0.14)
+- [Go 1.16.3](https://golang.org/doc/go1.16)
+- [Docker Compose 1.29.1](https://github.com/docker/compose/releases/tag/1.29.1)
+- [Docker Engine 20.10.6](https://docs.docker.com/engine/release-notes/#20106)
+
+## Bug fixes and minor changes
+
+- Fixed a bug with an Apple chip where the last byte in a network transfer was occasionally lost.
+- Fixed a bug where a `metrics-port` defined in the engine's `daemon.json` blocks application restart.
+- Fixed a leak dynamic ports ("ephemeral ports") e.g. `docker run -p 80`. Fixes [docker/for-mac#5611](https://github.com/docker/for-mac/issues/5611).
+- Emulate a more modern Intel CPU with `qemu` on an Apple chip, for better image compatibility. See [docker/for-mac#5561](https://github.com/docker/for-mac/issues/5561).
+- Removed the "Deploy Docker Stacks to Kubernetes by default" Kubernetes setting. Fixes [docker/for-mac#4966](https://github.com/docker/for-mac/issues/4966).
```

Comment:
I think we need to say more about this. It might not be clear why we removed an option and what people are supposed to do instead.

Did we relnote when we removed compose on kube? -- if so we could point to it. We should say that the option has been removed because c-o-k is no longer bundled but is still available for download separately. (If that's accurate).

Reply:
In 2.4.0.0 we said:

> - Compose-on-Kubernetes is no longer included in the Docker Desktop installer. You can download it separately from the compose-on-kubernetes [release page](https://github.com/docker/compose-on-kubernetes/releases).

The repo `docker/compose-on-kubernetes` has since been archived and is read-only.

Looking at the [referenced bug report](https://github.com/docker/for-mac/issues/4966) the situation is unclear:
> I have the same problem and now https://github.com/docker/compose-on-kubernetes is archived and no longer mantained.
> This is not well documented at all. From the documentation one would think that this would work out of the box

There is an open PR to remove a reference to it from the docs: https://github.com/docker/docker.github.io/pull/12737

I'm not sure what to recommend to users. I'll do a bit more digging.

Reply:
I've pushed a patch to recommend using https://kompose.io which has the closest workflow i.e.
- develop with `docker compose` locally as normal
- run `kompose convert` to generate some yaml
- `kubectl apply` to deploy to a cluster

I also moved this line to the top of the list. Maybe we should have a section for "Removed Features"?

Reply:
Yes, agree. We can add a new section below bug fixes called 'Removed features' and add this note there for visibility. We can do similar in upcoming releases for deprecated features too.

Reply:
I've added the "Removed features" section and moved the note into it. I removed the specific recommendation for kompose for now as I haven't tried it personally and am not 100% sure whether it's a suitable recommendation or not.